### PR TITLE
ISSUE-583: Set resources requests/limits to the init containers

### PR DESCRIPTION
**Pull request by mina-alber**
Fixing https://github.com/pegasystems/pega-helm-charts/issues/583

### CODE REVIEW

**Comment by efiku** [comment]
https://github.com/pegasystems/pega-helm-charts/tree/master/charts/pega#managing-resources

**Comment by mina-alber** [comment]
@efiku 

This works only for the application containers, but the init container doesn't have a property to adjust/specify its resources.

Bug/Issue details: https://github.com/pegasystems/pega-helm-charts/issues/583

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Review by MadhuriArugula — APPROVED** [review]

File: charts/pega/charts/installer/templates/_helpers.tpl [hunk 1]
```diff
@@ -95,6 +95,14 @@
       value: "{{ template "k8sWaitForWaitTime" $ }}"
     - name: MAX_RETRIES
       value: "{{ template "k8sWaitForMaxRetries" $ }}"
+  resources:
```
Comment (kishorv10) [inline charts/pega/charts/installer/templates/_helpers.tpl:98]: Can a reusable template be created for k8sWaitFor image resource settings and refer it to other places? Probably a template in pega chart _helpers.tpl?

File: charts/pega/charts/installer/templates/_helpers.tpl [hunk 2]
```diff
@@ -95,6 +95,14 @@
...
+    # CPU and Memory that the init containers request
+    requests:
+      cpu: 50m
+      memory: 64Mi
+    limits:
+      cpu: 50m
+      memory: 64Mi
```
Comment (kishorv10) [inline charts/pega/charts/installer/templates/_helpers.tpl:105]: Is any base reference or data available to conclude these values?

**Comment by kishorv10** [comment]
@mina-alber  Can you add helm template test cases for the changes? Tests are located [here](https://github.com/pegasystems/pega-helm-charts/tree/master/terratest), helm template tests uses [terratest](https://github.com/gruntwork-io/terratest) framework.

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

File: charts/pega/charts/installer/templates/_helpers.tpl (hunk 2 above)
Comment (mina-alber) [reply charts/pega/charts/installer/templates/_helpers.tpl]: Not exactly, so I set the limits to low values as it won't need much resources for the initContainers.

**Comment by mina-alber** [comment]
@kishorv10 
I've updated my PR following your recommendation and added a test for it, please review it.

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by mina-alber** [comment]
Hi @Saurabh-16 

I've updated the helpers accordingly!

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-585 validation on -> integ-eks

**Comment by Saurabh-16** [comment]
This PR is taken care by #622

**Pull request closed without merging**